owncloud/core
# No max version check for apps when using release channel git or daily

## Description
When installing owncloud from git or the daily release tar ball the max version requirement is no longer checked when installing/enabling apps

This will help with the development and release of apps in general

@patrickjahns as discussed long time back - we did go for the release channel and no env variable.

## Related Issue
- fixes #31518 

## How Has This Been Tested?
- enable an app with max version lower then 11.0
- see if being enabled

## Screenshots (if appropriate):

## Types of changes

## Checklist:
